### ISSUE TITLE
Manual Trakt linking does not work anymore

### ISSUE THREAD

**Issue by t0815** [japanesemediamanager/ShokoDesktop#542]
Hi,

i just noticed, that manual Trakt linking does not work anymore. The search function is working, but once you have to select a season and episode, the dropdowns are always empty and nothing can be selected.

![image](https://user-images.githubusercontent.com/10379828/27993618-de89b7bc-64ad-11e7-9872-e71c386e3fc9.png)

Server Log (Desktop log didnt generate any data for that action):

[2017-07-09.txt](https://github.com/japanesemediamanager/ShokoDesktop/files/1133711/2017-07-09.txt)

**Comment by da3dsoul** [comment]
Version or it doesn't happen...
I'm thinking of removing Trakt linking entirely and making it automatic.

**Comment by da3dsoul** [comment]
If on dailies, that notification will suffice, it hasn't changed much for a while

**Comment by t0815** [comment]
> Version or it doesn't happen...
I'm thinking of removing Trakt linking entirely and making it automatic.
> 

hmm, not sure if thats a good idea tho, since sometimes automatic assignment is wrong and you need to correct it manually.

Shoko- Desktop version is 3.7.0.4

**Comment by da3dsoul** [comment]
No, it'll always be identical to TvDB. There will still be a way to manually override it in ClientV2, but I'm not too concerned as I plan to improve the auto linking even more.

**Comment by da3dsoul** [comment]
Okay no idea if it still happens. A lot has changed since then.

**Comment by da3dsoul** [comment]
We'll leave this open and it can be checked on dailies when desktop is fixed (it won't connect as of yesterday)

**Comment by ElementalCrisis** [comment]
This automatic right @da3dsoul?
